app/views/parliaments/houses/members/a_to_z.html.haml

```diff
@@ -1,9 +1,9 @@
-- title("#{t('mp_plural')} - #{@parliament.date_range(date_format: '%Y')} #{t('parliament')} #{t('parliaments.a_to_z')}")
+- title("#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')} - #{@parliament.date_range(date_format: '%Y')} #{t('parliament')} #{t('parliaments.a_to_z')}")
 #content.section--primary{ tabindex: "0" }
   .container
     %h1
-      %span= "#{t('mp_plural')}"
-      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament').capitalize}", parliament_path(@parliament.graph_id))
+      %span= "#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')}"
+      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament')}", parliament_path(@parliament.graph_id))
```

Comment:
Do we still need to capitalise Parliament?

Reply:
Reviewed with Alan, who would like 'parliament' to stay lowercase across the whole site